<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 0);</string>
  </property>
  <widget class="QPushButton" name="pb_venta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(170, 0, 255);
background-color: rgb(255, 0, 0);
font: 75 14pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Vender Producto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Arial&quot;;
color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>Â¿Que deseas hacer?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ver">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(170, 0, 255);
background-color: rgb(255, 0, 0);
font: 75 14pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ver Ventas de Producto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_empleados">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-color: rgb(170, 0, 255);

font: 75 14pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ver Empleados</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
